.github/styles/Aiven/capitalization_headings.yml

```diff
@@ -6,3 +6,4 @@ scope: heading
 match: $sentence
 exceptions:
   - HowTo
+  - Paglia
```

Comment:
What is this doing here? The exceptions are needed for words that cannot be added to the dictionary, but don't meet the sentence case rules (HowTo has a second uppercase letter so it causes a problem).